documentation/Camara-OAS-Sections-Guidelines.md

```diff
@@ -0,0 +1,108 @@
+# CAMARA OAS Sections Guidelines 
+
+## Table of Contents
+
+   * [Introduction](#introduction)
+   * [Conventions](#conventions)
+   * [OAS Sections](#oas-sections)
+     + [OpenAPI Version](#version)
+     + [Info Object](#info-object)
+       + [Title](#title)
+       + [Description](#description)
+       + [Version](#version)
+       + [Terms of service](#tos)
+       + [Contact information](#contact)
+       + [License](#license)
+       + [Extension field](#ext-field)
+     + [Servers](#servers)
+     + [Paths](#paths)
+     + [Components](#components)
+       + [Schemas ](#schemes)
+       + [Responses](#responses)
+       + [Parameters](#parameters)
+       + [Request bodies](#req-bodies)
+       + [Headers](#headers)
+       + [Security schemes](#sec-schemes)
+     + [Security](#security)
+     + [Tags](#tags)
+     + [External Documentation](#ext-doc)
+
+## Introduction
+Camara uses OpenAPI Specification (OAS) to describe its APIs. The below guidelines specify the restrictions or conventions to be followed within the OAS yaml by all Camara APIs (referred below simply as APIs).
+
+## Conventions
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+## OAS Sections
+
+### OpenAPI Version
+APIs shall use the OAS version 3.0.3.
+
+### Info Object
+An example of the info object is shown below:
+```
+info:
+  title: Number Verification
+  description: |
+    This API allows to verify that the provided mobile phone number is the one used in the device. It
+    verifies that the user is using a device with the same mobile phone number as it is declared.
+  version: 1.0.1
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  x-camara-commonalities: 0.4.0
+  
+```
+#### Title
+Title describes the API shortly. The title shall not include the term "API" in it.
```

Comment:
I think "Title" should be the "formal" (or "friendly") name of the API, rather than a free text description. Ideally, this name would also documented by [API Backlog](https://github.com/camaraproject/APIBacklog/blob/main/documentation/APIbacklog.md), though this table would need to be updated as it is currently sub-project rather than API centric.

"Formal" names will be important for marketing purposes. Whilst we can be sure all implementations will use, for example, the name `/number-verification` within the path, we cannot be sure that the associated implementation documentation will use the name "Number Verification".